src/index.html

```diff
@@ -11,7 +11,170 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
-    <script type="text/javascript" src="./main.js"></script>
+
+    <form
+      action="http://localhost:300api"
+      method="get"
+    >
+      <fieldset>
+        <legend>Personal information</legend>
+        <label
+        class="form-field"
+        >
```

Comment:
there's no need to move attribute to the next line if it's the only attribute for tag and if line width is lower then 80 symbols, fix it everywhere
```suggestion
        <label class="form-field">
```